md/release-notes.md

```diff
@@ -84,7 +84,23 @@ Bonita binaries as well as Studio installer are now officially signed on Windows
 ### Form mapping API error messages
 Improvements of messages in case of error on form mapping API calls have been done to explain the cause of error and give options for easier error recoveries.
 
-<a id="techonolgy-updates"/>
+### Performance (applies to Bonita 7.7.5+)
+The following improvements have been made to Bonita performance:
+- There is less I/O requests to temp folders
+- Work execution consumes less database resources
+- The login REST APIs mechanism consumes less resources
+- At engine startup and BPM services resume, the process dependencies loading consumes less memory
+
+A special attention has also been carried out to an improved deletion mechanism of archived cases. Deleting archived cases is now way more efficient.
+
+:::warning
```

Comment:
I thought we were about deprecating and deleting those event handlers. 
So there will be no long term option of reactivating it back.
The value (or lack of value) of these events should be pointed here so the reader realizes there is no point of wanting them.
I have not synchronized with CS before writing that. If you have, then you words must be kept.

Reply:
When we have an agreement on that, this must be added to the 7.8.0 release notes too.

Reply:
It will be reported to 7.8.0 RN, as soon as CS confirmes your remark above.

Reply:
Checked and integrated review from CS team...